remoting/zookeeper/listener.go

```diff
@@ -247,8 +247,13 @@ func (l *ZkEventListener) listenDirEvent(conf *common.URL, zkPath string, listen
 			l.pathMapLock.Lock()
 			l.pathMap[dubboPath] = struct{}{}
 			l.pathMapLock.Unlock()
-
+			//When Zk disconnected, the Conn will be set to nil, so here need check the value of Conn
+			l.client.RLock()
+			if l.client.Conn == nil {
+				break
```

Comment:
Can this case trigger to auto-reconnect?

Reply:
I have the same question. But travis still failing.

Reply:
In my opinion, there is already have other case for trigger reconnect, so here may be just need stop the resubscribe process and wait reconnect finished.

By the way, the travis failed should not be caused by my changes, the error is:

![image](https://user-images.githubusercontent.com/11661906/84863191-27ae9000-b0a7-11ea-96d7-0ff4bb8eaa10.png)

may be you can re-trigger it..